.github/workflows/cgroup2.yaml

```diff
@@ -31,6 +31,17 @@ jobs:
       - name: Check out code
         uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
 
+      #fix https://github.com/kubernetes-sigs/kind/issues/3411
```

Comment:
Still needed? https://github.com/kubernetes-sigs/kind/issues/3411#issuecomment-1792060741

Reply:
The GitHub Action workflows have been reworked, so this may not even be relevant anymore.